radicle/src/storage/git.rs

```diff
@@ -413,6 +414,16 @@ impl Repository {
 
         Ok(refs)
     }
+
+    pub fn resolve_oid(&self, oid: cob::ObjectId) -> Result<cob::ObjectId, git2::Error> {
+        let mut object_id = oid.to_string();
+        if object_id.starts_with("00") {
+            object_id = object_id.trim_start_matches("00").to_owned();
```

Comment:
I think we can avoid `to_owned()` here since `revparse_single` takes &str?

Reply:
Mh yes and no, now with the as_bytes rework I think there is some borrow error, but I need to think about it :) but I think you are right, we can do better